### ISSUE TITLE
Question + feature requests

### ISSUE THREAD

**Issue by Cestis** [yuliskov/SmartTube#47]
Hi!
Question: does the app have an auto updater (similar to the one from smartyoutubetv)?

If not, I request that feature!
Also:
It would be nice, if the video overlay (with all quality, timeline etc. options) would disappear automatically after a few seconds instead of requiring to click back to close it.
Also:
It would be nice, if there was a quality preset so I can always run the best possible quality available.

**Comment by hussainalhaddad** [comment]
Yes, it does have an auto update feature.. even better than the previous version

**Comment by WolfganP** [comment]
Every time I open the app I see a new version was auto-installed (which I don't like). 
Is there any way to move back to previous behavior? (receive notifications but manual update actions?)
Thx

**Comment by foreveryoungforty** [comment]
> Every time I open the app I see a new version was auto-installed (which I don't like).
> Is there any way to move back to previous behavior? (receive notifications but manual update actions?)
> Thx

You can uncheck automatic update in the settings>about/info of the app.

**Comment by WolfganP** [comment]
> You can uncheck automatic update in the settings>about/info of the app.

Thanks, but when I did that I stopped receiving notifications as well. Is that a bug or the intended behavior?

**Comment by foreveryoungforty** [comment]
> > You can uncheck automatic update in the settings>about/info of the app.
> 
> Thanks, but when I did that I stopped receiving notifications as well. Is that a bug or the intended behavior?

That's normal behavior , because auto update unchecked means of course that you have to control for update manually.

**Comment by WolfganP** [comment]
> That's normal behavior , because auto update unchecked means of course that you have to control for update manually.

Understood, thx for the confirmation. If there's a chance, it may be wonderful to have the functionality separated to have more control.
